### DISCUSSION TITLE
MDX_Tool_1.7

### DISCUSSION THREAD

**Discussion by BobanSpasic** [BobanSpasic/MDX_Tool#6] [Announcements]
New tool added - MDX_Collect

<hr /><em>This discussion was created from the release <a href='https://github.com/BobanSpasic/MDX_Tool/releases/tag/1.7.0.40'>MDX_Tool_1.7</a>.</em>

**Comment by BobanSpasic** [comment]
Dang, there is still a problem with hashing the voices - I get different hashes for a voice inside a bank (VMEM), and for the same voice saved as a single voice (VCED). It worked fine at one point, but it is wrong again, and it got into the release with this bug.

**Comment by josbouten** [comment]
I'd like to give this code a try, but I will need an executable or know how to compile it. I've worked with pascal 40 odd years ago. And I'm not fluent in how to compile pascal code on linux these days. Can you give me some pointers or point me to a binary?
